start.py

```diff
@@ -44,5 +44,11 @@ def start_jenkins():
 
 
 if __name__ == "__main__":
-    proc = start_jenkins()
-    sys.exit(proc.wait())
+    proc = None
+
+    try:
+        proc = start_jenkins()
```

Comment:
Maybe you can put this before the try ? You wouldn't have to handle the case when proc is None.